### ISSUE TITLE
IJulia install error

### ISSUE THREAD

**Issue by stardiviner** [JuliaLang/IJulia.jl#379]
Here is the output:
```
Writing IJulia kernelspec to julia-0.4/kernel.json ...
Installing julia kernelspec julia-0.4
Traceback (most recent call last):
  File "/home/stardiviner/.virtualenvs/python3/bin/ipython", line 11, in <module>
    sys.exit(start_ipython())
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/IPython/__init__.py", line 118, in start_ipython
    return launch_new_instance(argv=argv, **kwargs)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/traitlets/config/application.py", line 591, in launch_instance
    app.initialize(argv)
  File "<decorator-gen-111>", line 2, in initialize
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/traitlets/config/application.py", line 75, in catch_config_error
    return method(app, *args, **kwargs)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/IPython/terminal/ipapp.py", line 305, in initialize
    super(TerminalIPythonApp, self).initialize(argv)
  File "<decorator-gen-7>", line 2, in initialize
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/traitlets/config/application.py", line 75, in catch_config_error
    return method(app, *args, **kwargs)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/IPython/core/application.py", line 386, in initialize
    self.parse_command_line(argv)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/IPython/terminal/ipapp.py", line 300, in parse_command_line
    return super(TerminalIPythonApp, self).parse_command_line(argv)
  File "<decorator-gen-4>", line 2, in parse_command_line
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/traitlets/config/application.py", line 75, in catch_config_error
    return method(app, *args, **kwargs)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/traitlets/config/application.py", line 487, in parse_command_line
    return self.initialize_subcommand(subc, subargv)
  File "<decorator-gen-3>", line 2, in initialize_subcommand
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/traitlets/config/application.py", line 75, in catch_config_error
    return method(app, *args, **kwargs)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/traitlets/config/application.py", line 418, in initialize_subcommand
    subapp = import_item(subapp)
  File "/home/stardiviner/.virtualenvs/python3/lib/python3.5/site-packages/ipython_genutils/importstring.py", line 31, in import_item
    module = __import__(package, fromlist=[obj])
ImportError: No module named 'jupyter_client'
===============================[ ERROR: IJulia ]================================

LoadError: could not spawn `ipython-kernelspec install --replace --user julia-0.4`: no such file or directory (ENOENT)
while loading /home/stardiviner/Git/dotfiles/.julia/v0.4/IJulia/deps/build.jl, in expression starting on line 109

================================================================================

================================[ BUILD ERRORS ]================================

WARNING: IJulia had build errors.

 - packages with build errors remain installed in /home/stardiviner/Git/dotfiles/.julia/v0.4
 - build the package(s) and all dependencies with `Pkg.build("IJulia")`
 - build a single package by running its `deps/build.jl` script

================================================================================
INFO: Package database updated
```

**Comment by stevengj** [comment]
It sounds like you haven't fully installed jupyter.

**Comment by stevengj** [comment]
(If you're not sure how to install jupyter, you can always ask Julia to install its own by doing `ENV["JUPYTER"]=""; Pkg.build("IJulia")`)

**Comment by gomiero** [comment]
Hi, @stardiviner!

I had the same problem that you reported on my system: an Windows 8.1 x64 system, with Julia 0.4.0, Python 2.7 and Jupyter installed (and working), and the environment variable PATH configured  ok for Julia dir,  Python dir and Python/Scripts dir (where ipython and jupyter are located).

To install IJulia with sucess, I opened the file *%YOUR_HOME%/.julia/v0.4/IJulia/deps/build.jl* in a text editor and I changed the line 135 of this file 

from:
```
run(`$python $jk_path install --replace --user $juliakspec`)
```

to:
```
run(`$python $jk_path install --replace --user $(pwd())\\$juliakspec`)
```

(The **\\\\** double slash is because of Windows path separator)

After this change, I executed the command (on Julia REPL):

```
julia> Pkg.build("IJulia")
```

and everything works fine.

Hope this helps!

Sorry if I made mistakes with the English language. :-)
